README.md

```diff
@@ -25,6 +25,12 @@ Finally, inside your Ruby program, load the Nokogiri gem and start parsing your
 
 For more information about how to use RubyGems, see our RubyGems basics guide at [guides.rubygems.org](http://guides.rubygems.org/rubygems-basics/)
 
+## Requirements
+
+* RubyGems 2.6 support Ruby 2.4 or lower version of Ruby.
+* RubyGems 2.7 support Ruby 2.5 or lower version of Ruby.
+* RubyGems 3.0 support Ruby 2.3+.
```

Comment:
I'm not great at English but I think "RubyGems 2.6 supports Ruby 2.4 or lower" reads better.